src/lib/Scenes/Home/Components/Sales/Components/LotsByFollowedArtists.tsx

```diff
@@ -0,0 +1,126 @@
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import React, { Component } from "react"
+import createEnvironment from "lib/relay/createEnvironment"
+import renderWithLoadProgress from "lib/utils/renderWithLoadProgress"
+import styled from "styled-components/native"
+import { Button, ScrollView, Text, View } from "react-native"
+import { RelayPaginationProp } from "react-relay"
+import { SectionHeader } from "./SectionHeader"
+import { createPaginationContainer, graphql, QueryRenderer, QueryRendererProps } from "react-relay"
+import { get, once } from "lodash"
+import { isCloseToBottom } from "lib/utils/isCloseToBottom"
+
+const DEFAULT_TITLE = "Lots by Artists You Follow"
+const PAGE_SIZE = 8 // FIXME: Increase
+
+interface RelayProps {
+  relay: RelayPaginationProp
+  section: {
+    title?: string
+  }
+  viewer: {
```

Comment:
we should add `| null` for the type defs as `viewer` can be null, nested properties as well